library/alloc/src/vec.rs

```diff
@@ -2578,7 +2578,7 @@ impl<T: PartialOrd> PartialOrd for Vec<T> {
 #[stable(feature = "rust1", since = "1.0.0")]
 impl<T: Eq> Eq for Vec<T> {}
 
-/// Implements ordering of vectors, lexicographically.
+/// Implements ordering of vectors, [lexicographically](../../std/cmp/trait.Ord.html#lexicographical-comparison).
```

Comment:
```suggestion
/// Implements ordering of vectors, [lexicographically](core::cmp::Ord#lexicographical-comparison).
```
Similar for the rest of the links.